### PR TITLE
Mark staff domains as authorised

### PR DESCRIPTION
**Pull request by andrewstech**
STAFF_MEMBERS env should be like. andrew, william, val

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -102,9 +102,16 @@ async function uploadImageToImgbb(image) {
   }
 }
 
+async function isStaffMember(username) {
+  const staffMembers = process.env.STAFF_MEMBERS.split(",");
```
Comment (wdhdev) [inline utils.js:106]: Can you make it get users from the org instead of from the env file. Only include members in the `@is-a-dev/maintainers` team.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: utils.js (hunk 1 above)
Comment (andrewstech) [reply utils.js:106]: that would require a user with a pat token @is-a-dev-bot should work however im not sure if it would have to be on the team.

Comment (wdhdev) [reply utils.js:106]: @andrewstech The bot is in the organisation so it should be fine.

Comment (andrewstech) [reply utils.js:106]: Okay lol need to do this

**Pull request closed without merging**